includes/class-migration.php

```diff
@@ -272,6 +272,33 @@ private static function migrate_from_2_6_0() {
 		self::update_options_key( 'activitypub_blog_user_identifier', 'activitypub_blog_identifier' );
 	}
 
+	/**
+	 * Update actor-mode settings
+	 *
+	 * @return void
+	 */
+	private static function migrate_from_4_0_0() {
+		$blog_profile    = \get_option( 'activitypub_enable_blog_user', false );
+		$author_profiles = \get_option( 'activitypub_enable_users', false );
+
+		if (
+			'1' === $blog_profile &&
+			'1' === $author_profiles
+		) {
+			\update_option( 'activitypub_actor_mode', '3' );
```

Comment:
I think we should define some constants somewhere for these values rather than hardcoding them like this.